public/index.html

```diff
@@ -0,0 +1,370 @@
+                <img src="./imgs/hexagon.jpg" alt="welcome sign">
+                <p><strong>Hexagon onepage</strong>
+                    <br>Web design, brand &#38; logotypes</p>
+            </li>
+            <li class="flex-item-2">
+                <img src="./imgs/sketches.jpg" alt="sketches of fantasy characters">
+                <p><strong>Sketches icon</strong>
+                    <br>Icons design, brand &#38; logotypes</p>
+            </li>
+            <li class="flex-item-2">
+                <img src="./imgs/logotypes.jpg" alt="screen with various logos on cards">
+                <p><strong>Logotypes volume</strong>
+                    <br>UI design, brand &#38; logotypes</p>
+            </li>
+            <li class="flex-item-2">
+                <img src="./imgs/flight.jpg" alt="screen with car and poitns">
+                <p><strong>Flight control</strong>
+                    <br>Webdesign, brand &#38; logotypes</p>
+            </li>
+            <li class="flex-item-2">
+                <img src="./imgs/ytube.jpg" alt="screen displaying ytube app">
+                <p><strong>Youtube app</strong>
+                    <br>Icons design, brand &#38; logotypes</p>
+            </li>
+            <li class="flex-item-2">
+                <img src="./imgs/management.jpg" alt="screen displaying claim management site">
+                <p><strong>Management</strong>
+                    <br>UI design, brand &#38; logotypes</p>
+            </li>
+            <li class="flex-item-2">
+                <img src="./imgs/koreanbeef.jpg" alt="screen displaying korean beef recipe">
+                <p><strong>Korean beef</strong>
+                    <br>Webdesign, brand &#38; logotypes</p>
+            </li>
+            <li class="flex-item-2">
+                <img src="./imgs/thumbs.jpg" alt="screen displaying thumbs up and down icons">
+                <p><strong>Facebook thumbs</strong>
+                    <br>Icons design, brand &#38; logotypes</p>
+            </li>
+        </ul>
+    </section>
+    <section class="section-black">
+        <object class="clip" type="image/svg+xml" data="./imgs/clipwhite.svg"></object>
+        <a id="about"></a>
+        <h1>ABOUT</h1>
+        <div class="column-two">
+            <p class="column-left">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+            <p class="column-right">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+        </div>
+        <h2 class="subheader subheader-skills">SKILLS</h2>
+        <p class="skill-list">PHOTOSHOP</p>
+        <progress value="78" max="100"></progress>
```

Comment:
Good use of `progress`, but I think the `progress` element and it's associated `p` should be wrapped in a`figure`/`figcaption` structure.